internal/scaffold/ansible/molecule_test_local_playbook.go

```diff
@@ -44,115 +44,34 @@ const moleculeTestLocalPlaybookAnsibleTmpl = `---
 - name: Build Operator in Kubernetes docker container
   hosts: k8s
   vars:
-    image_name: [[.Resource.FullGroup]]/[[.ProjectName]]:testing
+    image: [[.Resource.FullGroup]]/[[.ProjectName]]:testing
   tasks:
-  # using command so we don't need to install any dependencies
-  - name: Get existing image hash
-    command: docker images -q {{ image_name }}
-    register: prev_hash
-    changed_when: false
-
-  - name: Build Operator Image
-    command: docker build -f /build/build/Dockerfile -t {{ image_name }} /build
-    register: build_cmd
-    changed_when: not prev_hash.stdout or (prev_hash.stdout and prev_hash.stdout not in ''.join(build_cmd.stdout_lines[-2:]))
+    # using command so we don't need to install any dependencies
+    - name: Get existing image hash
+      command: docker images -q {{ image }}
+      register: prev_hash_raw
+      changed_when: false
+
+    - name: Build Operator Image
```

Comment:
I would label this "Build Operator image if necessary." (or "if changed")... because it doesn't always build the image.